### DISCUSSION TITLE
[BUG]

### DISCUSSION THREAD

**Discussion by DOGEEER6** [GameServerManagers/LinuxGSM#4082] [General]
When I start the server, it shows this
I need help!

$ ./csgoserver start
[ FAIL ] Starting csgoserver: executable was not found
* /home/dpcsgo/serverfiles/srcds_run

**Comment by Claiyc** [comment]
Did you successfully run the install command?

**Comment by DOGEEER6** [comment]
> 

Yes

**Comment by Claiyc** [comment]
What is the content of /home/dpcsgo/serverfiles?

**Comment by DOGEEER6** [comment]
> 

Two folders named "csgo" and "steamapps"

**Comment by Claiyc** [comment]
Idk what the folder structure of csgo is supposed to look like. You can try ./csgoserver validate
